pcs/prop.py

```diff
@@ -161,6 +164,18 @@ def list_property(lib, argv, modifiers):
         print(" {0}: {1}".format(prop, val))
 
 
+def get_default_property(property_name):
+    """
+    Commandline options: no options
+    """
+    parameter = {}
+    prop_def_dict = utils.get_cluster_properties_definition()
+    for name, prop in prop_def_dict.items():
+        if name == property_name:
+            parameter[name] = prop["default"]
+    return parameter
```

Comment:
This function is the same as existing function `get_default_properties`, except it filters the result. It's better to use the existing function and filter its result to avoid code duplication.